cli/compose/schema/data/config_schema_v3.8.json

```diff
@@ -96,6 +96,7 @@
         },
         "cap_add": {"type": "array", "items": {"type": "string"}, "uniqueItems": true},
         "cap_drop": {"type": "array", "items": {"type": "string"}, "uniqueItems": true},
+        "cgroupns_mode": {"type": "string"},
```

Comment:
Oh! With Docker 19.03 being released (using compose schema 3.8), this version of the schema is now frozen, so this will have to be moved to the 3.9 schema.

I just opened https://github.com/docker/cli/pull/2073 to add the 3.9 schema (currently a copy of the 3.8 schema); once that's merged, can you rebase your pull-request, and move these changes to the v3.9 schema?

(sorry for the hassle 🤗)

Reply:
https://github.com/docker/cli/pull/2073 was merged, could you update this PR?

Reply:
Updated - thanks for the ping!